### PR TITLE
Update dependencies list WIP

### PR DESCRIPTION
**Pull request by juanarbol**
`http-parser` was removed [see](https://github.com/nodejs/node/pull/29589), I just updated dependencies file

### CODE REVIEW

File: locale/en/docs/meta/topics/dependencies.md [hunk 1]
```diff
@@ -39,13 +39,13 @@ asynchronously at the operating system level.
 
 - [Documentation](http://docs.libuv.org/)
 
-### http-parser
+### llhttp
 
-HTTP parsing is handled by a lightweight C library called http-parser. It is
+HTTP parsing is handled by a lightweight C library called llhttp. It is
```
Comment (Trott) [inline locale/en/docs/meta/topics/dependencies.md]:
```suggestion
HTTP parsing is handled by a lightweight TypeScript and C library called llhttp. It is
```

You'll need to word-wrap to 80 chars if you accept this change.

Comment (juanarbol) [reply locale/en/docs/meta/topics/dependencies.md]: Thanks for reviewing! Fixed at 6be662d

**Comment by Trott** [comment]
This LGTM but a 👍 from @indutny to make sure we've got it exactly right (especially the removing http-parser part) would be great.

**Review by MylesBorins — APPROVED** [review]
LGTM

**Pull request merged**